docker-build/Dockerfile.cloud

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2022 Eugene Brodsky (https://github.com/ebr)
```

Comment:
@lstein: is this copyright compatible with the repositorys license?

Reply:
same as above - removed